gitman/models/config.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=attribute-defined-outside-init
```

Comment:
I'd rather we not disable this warning and set all attributes in `__init__` (to `None` if needed).